src/app/test/app_test.mocks.dart

```diff
@@ -0,0 +1,106 @@
+// Mocks generated by Mockito 5.4.4 from annotations
+// in app/test/app_test.dart.
+// Do not manually edit this file.
+
+// ignore_for_file: no_leading_underscores_for_library_prefixes
+import 'dart:async' as _i3;
+
+import 'package:app/business/dad_jokes/dad_joke.dart' as _i5;
+import 'package:app/business/dad_jokes/dad_jokes_service.dart' as _i4;
+import 'package:app/business/diagnostics/diagnostics_service.dart' as _i2;
+import 'package:mockito/mockito.dart' as _i1;
+
+// ignore_for_file: type=lint
+// ignore_for_file: avoid_redundant_argument_values
+// ignore_for_file: avoid_setters_without_getters
+// ignore_for_file: comment_references
+// ignore_for_file: deprecated_member_use
```

Comment:
est-ce que je devrais ajouter dans le gitignore? @Soap-141

Reply:
Maybe all files that ends with `.mocks.dart`

Reply:
We could do it, but if you choose to, make sure that the required commands are run before the Pipeline execute tests and all.

Reply:
@jeanplevesque

Reply:
It should but just make sure that it's all good.